### ISSUE TITLE
"Power" devices of different unit_of_measurements are merged

### ISSUE THREAD

**Issue by syphernl** [jseidl/hass-magic_areas#54]
Some of my plugs report multiple values (e.g. W, kWh, A, V etc) but currently they are all being grouped in the `sensor.area_power_*` sensors. It would be better if they could be grouped in `unit_of_measurement`

```
HASS Version | 0.118.3
HASS Type | Home Assistant Core
Magic_Areas version | 1.1.0
```

**Comment by KTibow** [comment]
Could you clarify? Do you mean grouping plugs with same units of measurement?

**Comment by syphernl** [comment]
> Could you clarify? Do you mean grouping plugs with same units of measurement?

Yes. One of my plugs exposes 5 entities with different units of measurements: 
- sensor.zolder_plug_wasdroger_power
- sensor.zolder_plug_wasdroger_electricity_a_meter 
- sensor.zolder_plug_wasdroger_electricity_kwh_meter (counter of used kwh)
- sensor.zolder_plug_wasdroger_electricity_pf_meter
- sensor.zolder_plug_wasdroger_electricity_v_meter

They are however being combined into one `*_power` entity by magic_areas which does not make much sense to me with the different units. 
For me, only the first one with `unit_of_measurement: W` is interesting. I could add them to the exclude (which I have done) but perhaps the "out of the box" behavior could be tweaked a little so it would make more sense?

**Comment by KTibow** [comment]
Any reason you couldn't just disable them from the UI in the integration page?

**Comment by jseidl** [comment]
The way that currently aggregation works is by seeing which `unit_of_measurement` pops out the most and use it. It's a wild guess. Division by `unit_of_measurement` would create a lot of complexity. 

One solution for temperature would be checking the unit of measurement of the HA installation and maybe for power only consider watts.

**Comment by jseidl** [comment]
This is being fixed on #74. Aggregations now consider not only `device_class` but also `unit_of_measurement` which will give different `power` device class aggregates for each `unit_of_measure`. Coming up on release 2.0.0

**Comment by jseidl** [comment]
Closed by #74, released on [2.0.0](https://github.com/jseidl/hass-magic_areas/releases/tag/v2.0.0).
